src/ops/inv.rs

```diff
@@ -0,0 +1,29 @@
+/// Unary operator for retrieving the multiplicative inverse, or reciprocal, of a value.
+pub trait Inv {
+    /// The result after applying the operator.
+    type Output;
+
+    /// Returns the multiplicative inverse of `Self`.
+    fn inv(self) -> Self::Output;
+}
+
+macro_rules! inv_impl {
+    ($t:ty, $out:ty, $fn:expr) => {
+        impl<'a> Inv for $t {
+            type Output = $out;
+
+            #[inline]
+            fn inv(self) -> $out {
+                ($fn)(self)
```

Comment:
I don't this function call will actually work for the reference versions, and it's weird having a lifetime parameter on the value versions.  There are only 4 implementations here, so maybe it's not worth trying to make a unified macro.

To avoid `cfg(feature = "std")`, we could just implement these as `1.0 / self`.  That's all the standard library is doing anyway.

Reply:
I'll do that later today